linera-service/tests/end_to_end_tests.rs

```diff
@@ -1308,7 +1310,7 @@ async fn test_end_to_end_fungible() {
     let value = app1
         .get_fungible_account_owner_amount(&account_owner2)
         .await;
-    assert_eq!(value, Amount::from(3));
```

Comment:
for the control check, maybe best would be a function taking an application and a btreemap. that would be nicer and easier to read.

Reply:
In tests, the code is usually more streamlined (less intermediate functions) precisely to make it easier to remove

Reply:
Just to be clear: for me, the current code is fine here. Another way to write this assertion (also fine) is:
```
assert_eq!(
    app1.get_fungible_account_owner_amount(&account_owner2).await,
    Amount::from(3),
);
```